<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>energyDialog</class>
 <widget class="QDialog" name="energyDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>463</width>
    <height>156</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="EgretBox">
     <property name="minimumSize">
      <size>
       <width>455</width>
       <height>100</height>
      </size>
     </property>
     <property name="title">
      <string>Energy Range</string>
     </property>
     <widget class="QWidget" name="layoutWidget_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>230</width>
        <height>56</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QFormLayout" name="minLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="minLabel">
           <property name="text">
            <string>Minimum Energy</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="minEnergyEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QFormLayout" name="maxLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="maxLabel">
           <property name="text">
            <string>Maximum Energy</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="maxEnergyEdit"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="minimumSize">
        <size>
         <width>240</width>
         <height>30</height>
        </size>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Help|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
